modules/modify-data-object/build/modify-data-object/Dockerfile

```diff
@@ -8,10 +8,11 @@ ENV PATH=$PATH:/usr/local/go/bin
 COPY . .
 RUN CGO_ENABLED=0 GOOS=linux go build -o /${TASK_NAME} cmd/${TASK_NAME}/main.go
 
-FROM registry.access.redhat.com/ubi9/ubi-minimal:latest
+FROM quay.io/centos/centos:stream9
```

Comment:
Is centos:stream9 used by every task? Why change it?

Reply:
ubi-minimal does not contain xorriso